controllers/shooter.py

```diff
@@ -34,19 +34,27 @@ def __init__(self) -> None:
 
     @state(first=True, must_finish=True)
     def preparing_intake(self) -> None:
+        self.tilt_component.goto_pre_intake()
         self.turret_component.set_angle(0.0)
-        self.tilt_component.goto_intaking()
 
-        if self.turret_component.at_angle() and self.tilt_component.at_angle():
+        if self.turret_component.at_angle():
```

Comment:
I think we still want to make sure the tilt component is in the correct position before we deploy the intake.
```suggestion
        if self.turret_component.at_angle() and self.tilt_component.at_angle():
```